registry/storage/driver/testsuites/testsuites.go

```diff
@@ -269,7 +269,7 @@ func (suite *DriverSuite) TestWriteReadStreamsNonUTF8(c *check.C) {
 	suite.writeReadCompareStreams(c, filename, contents)
 }
 
-// TestWriteReadLargeStreams tests that a 5GB file may be written to the storage
```

Comment:
ok.. the other option would be to warn them to run with a ton of RAM and we can increase the default time limit from 10min to a few hours for the non short case.